Formula/m/mubeng.rb

```diff
@@ -18,8 +18,8 @@ class Mubeng < Formula
   depends_on "go" => :build
 
   def install
-    ldflags = "-s -w -X github.com/kitabisa/mubeng/common.Version=#{version}"
-    system "go", "build", *std_go_args(ldflags:), "./cmd/mubeng"
+    ldflags = "-s -w -X github.com/mubeng/mubeng/common.Version=#{version}"
```

Comment:
```suggestion
    ldflags = "-s -w -X github.com/mubeng/mubeng/common.Version=v#{version}"
```

Upstream sets this via `git describe --always --tags`, and the upstream tags are `v`-prefixed. Prefixing our version here would stay consistent with upstream.

Reply:
Great catch! Thank you @alebcay 👍